presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -89,9 +89,9 @@
 import static com.google.common.util.concurrent.Futures.immediateFuture;
```

Comment:
I included it here because the end goal of this PR is to reduce this number. I had some measurement, but I forgot to put the result in the description of this PR.